Assets/Scripts/Wind.cs

```diff
@@ -0,0 +1,66 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Wind : MonoBehaviour
+{
+    private WindIndicator windIndicator;
+    private WindDirection[] windDirections = { WindDirection.LEFT,
+        WindDirection.RIGHT };
+
+    private WindDirection currentWindDirection;
+    private int currentWindStrength;
+
+    public int maxWindStrength;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        windIndicator = GetComponentInChildren<WindIndicator>();
+        updateIndicator();
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        foreach (GameObject bomb in getBombs())
+        {
+            applyForceToBomb(bomb);
+        }
+    }
+
+    public void BombDetonationEvent()
+    {
+        updateIndicator();
+    }
+
+    private void updateIndicator()
+    {
+        refreshWindValues();
+        windIndicator.UpdateWindIndicator(currentWindDirection, currentWindStrength);
+    }
+
+    private void refreshWindValues()
+    {
+        currentWindStrength = Random.Range(1, maxWindStrength + 1);
```

Comment:
I think we could include a no wind state, that could be cool and an opportunity for the player to feel smart by waiting for the lull in the wind